doc/core.md

```diff
@@ -142,7 +147,12 @@ Postcomputations cannot access stream values, but we can assume that all reducti
 What about "numflips" or a sum over all time?
 Since it's simply a reduce over all previous data, and the previous data isn't going to change, we should be able to just use the previous reduce value etc.
 
-So in summary, each fold reduction should output its current value as the lookbehind, and each latest reduction should output the values it used.
+There are three types of reductions: 
+
+- latest N: the used facts are stored
```

Comment:
Not sure if it worth differentiating between `latest n` and `latest where <some condition>` (or is that what the N is for here?). This latest where case is interesting complicated because `<some condition>` might be something like `first`. That sounds a bit weird reading it back - but I think it makes sense - grab me tomorrow if it doesn't.